cumulusci/cli/cci.py

```diff
@@ -602,14 +620,13 @@ def callback(ctx, project=False, *args, **kwargs):
 @click.command(
     cls=ConnectServiceCommand, name="connect", help="Connect a CumulusCI task service"
```

Comment:
It's working-ish since ctx.ensure_object(CliConfig) does about the same thing as load_config, but good catch, I'll update it to call load_config explicitly since we don't need it set on the ctx.